bookwyrm/templates/snippets/fav_button.html

```diff
@@ -3,18 +3,17 @@
 {% with status.id|uuid as uuid %}
 <form name="favorite" action="/favorite/{{ status.id }}" method="POST" class="interaction fav-{{ status.id }}-{{ uuid }} {% if request.user|liked:status %}is-hidden{% endif %}" data-id="fav-{{ status.id }}-{{ uuid }}">
     {% csrf_token %}
-    <button class="button is-small" type="submit">
-        <span class="icon icon-heart" title="{% trans 'Like status' %}">
-            <span class="is-sr-only">{% trans "Like status" %}</span>
+    <button class="button is-small is-light is-transparent" type="submit">
+        <span class="icon icon-heart m-0-mobile" title="{% trans 'Like' %}">
```

Comment:
These are still showing the text on mobile for me:
<img width="239" alt="Screen Shot 2021-04-24 at 11 25 28 AM" src="https://user-images.githubusercontent.com/1807695/115969045-d99cc100-a4ef-11eb-9a3f-1e34e8bfd28a.png">

Reply:
Strange, that's how it's displayed for me (tested on the latest Chromium, Firefox and Safari on macOS)

![Capture d’écran 2021-04-25 à 20 37 52](https://user-images.githubusercontent.com/1781070/115969385-f7444780-a53c-11eb-9a3c-12ffdafc775e.png)

Reply:
Oh that's my bad! I didn't run `collectstatic`. It looks perfect, nevermind :)